document/src/main/resources/config/liquibase/changelog/20221108-add-tenant-id.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="WouterG" id="1667899385-1">
+        <addColumn tableName="json_schema_document">
+            <column name="tenant_id" type="VARCHAR(256)">
+                <constraints nullable="false" />
```

Comment:
@WouterG moeten dit niet wel nullable worden. Ik het geval wij deze code willen mergen into valtimo. Op dit moment is een impl zelf verantwoordelijk voor het vullen van json_schema_document tabel met een tenant waarde. 
Omdat dit in de impl wordt verzorgd kunnen we nu niet dit op nullable false zetten denk ik.